### PR TITLE
doc: add description about memcached table

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
Follow-up https://github.com/groonga/groonga/issues/453

Due to my machine disk size problem, I can't update *.po files on my machine.
I'll appreciate if you create *.po files.

### CODE REVIEW

**Comment by hiroyuki-sato** [comment]
@kenhys @kou I made po file.
But I don't know how to write highlight text.

in rst file
```rst
`Memcache`
```

How to write po file?
```
\"memcache\"
```

Thanks!.

**Comment by kenhys** [comment]
Basically, describe as same as msgid for highlight.

For example, describe like

```
テーブルの名前は `Memcache` です。
```
in msgstr. Note that space is needed for keyword -  `Memcache` in this case.

**Comment by hiroyuki-sato** [comment]
@kenhys.

I fixed. Please check.

I confused how to highlight text.
Because this page has two highlight text.
http://groonga.org/ja/docs/server/memcached.html

One is 
```
`--protocol`
```

The other is 
```
"memcached"
```

**Comment by hiroyuki-sato** [comment]
I branched old tree. so I re-create pr with latest tree.

**Comment by kou** [comment]
@kenhys Could you check this?

@cosmo0920 Could you help @kenhys as an English professional?

File: doc/locale/ja/LC_MESSAGES/server.po [hunk 1]
```diff
@@ -449,6 +449,13 @@ msgstr ""
 "`--protocol` オプションとその引数により、サーバのプロトコルを指定することがで"
 "きます。\"memcached\"はmemcachedバイナリプロトコルを示しています。"
 
+msgid ""
+"You don't need to create table.  When groonga receive request, it create "
```
Comment (cosmo0920) [inline doc/locale/ja/LC_MESSAGES/server.po]: `table` is a countable noun. It need to add "a" here.
And `request` is also a countable. Ditto.

Comment (cosmo0920) [inline doc/locale/ja/LC_MESSAGES/server.po]: `it` points to `groonga`, right?
Then, it should add "s" like: "it create**s**"

File: doc/locale/ja/LC_MESSAGES/server.po [hunk 2]
```diff
@@ -449,6 +449,13 @@ msgstr ""
...
+"table automatically.  The table name is `Memcache` ."
```
Comment (cosmo0920) [inline doc/locale/ja/LC_MESSAGES/server.po]: This table doesn't exist before accepting request?
If so, we should use "future tense" like this: This table will be `Memcache`.

File: doc/locale/ja/LC_MESSAGES/server.po (hunk 1 above)
Comment (cosmo0920) [inline doc/locale/ja/LC_MESSAGES/server.po]: Be careful this `when clause` does not use "future tense" together. XD

**Comment by cosmo0920** [comment]
I've commented several nitpicks about tense, thanks.

**Comment by kenhys** [comment]
I've missed updated comment. thanks @hiroyuki-sato, @cosmo0920.

**Comment by kenhys** [comment]
> I confused how to highlight text.
> Because this page has two highlight text.
> http://groonga.org/ja/docs/server/memcached.html

Good catch, it may not be clear how to markup text for highlight. 
OT: I think that it is better to write a note for this topic in future. 
http://groonga.org/ja/docs/contribution/documentation.html

**Comment by hiroyuki-sato** [comment]
@cosmo0920 
Thanks.

I modified description. Could you check this?

You don't need to create a table.  Groonga creates a table automatically. The table will be `Memcache` .

**Comment by cosmo0920** [comment]
> You don't need to create a table.

:ok: 

> Groonga creates a table automatically.

"When groonga receives a request, it creates table automatically. " is more good sentence.

> The table will be Memcache .

"The table name will be `Memcache`."  is prefer. see: https://github.com/groonga/groonga/pull/454#discussion_r50351291

**Comment by hiroyuki-sato** [comment]
@cosmo0920 Thanks! fixed.

> You don't need to create a table. When groonga receives a request, it creates a table automatically. The table name will be `Memcache` .

I'll update *.po file later.

**Comment by kou** [comment]
(Please use "Groonga" instead of "groonga"...)

**Comment by hiroyuki-sato** [comment]
@kou 
Thanks, Changed.

> You don't need to create a table. When Groonga receives a request, it creates a table automatically. The table name will be `Memcache` .

**Comment by kenhys** [comment]
Merged. 
@hiroyuki-sato, thanks!

**Pull request merged**